modules/ROOT/pages/cloudhub-setup.adoc

```diff
@@ -26,10 +26,11 @@ The metrics that Anypoint Visualizer displays come from Anypoint Monitoring. As
 * Mule 4.1.x patch releases available on March 22, 2019 or later.
 * Mule 4.2.x.
 
-To see your app and the related metrics in Anypoint Visualizer, you must also enable the Monitoring agent after you upgrade as described in the next section.
+To see your app, its connections, and the related metrics in Anypoint Visualizer, you must also enable the Monitoring agent after you upgrade as described in the next section.
```

Comment:
Never abbreviate Anypoint Monitoring. If this is not Anypoint monitoring, then use common nouns: the monitoring agent.
```suggestion
To see your app, its connections, and the related metrics in Anypoint Visualizer, enable the Anypoint Monitoring agent after you upgrade.
```
GLOBALLY remove references to sections, topics, and so forth in the same deliverable, assuming that our topics are independent and reusable.